Interface/DragoonHudWindow.cs

```diff
@@ -0,0 +1,102 @@
+﻿using System;
+using System.Numerics;
+using Dalamud.Game.ClientState.Structs.JobGauge;
+using Dalamud.Plugin;
+using ImGuiNET;
+
+namespace DelvUIPlugin.Interface
+{
+    public class DragoonHudWindow : HudWindow
+    {
+        public override uint JobId => 32;
+
+        private new int BarHeight => 13;
+        private new int BarWidth => 254;
+        private new int XOffset => 127;
+        private new int YOffset => 466;
+
+        private readonly int xPadding = 2;
+
+        public DragoonHudWindow(DalamudPluginInterface pluginInterface, PluginConfiguration pluginConfiguration) : base(pluginInterface, pluginConfiguration) { }
+
+        protected override void Draw(bool _)
+        {
+            DrawHealthBar();
+            DrawPrimaryResourceBar();
+            DrawSecondaryResourceBar();
+            DrawTargetBar();
+        }
+
+        protected override void DrawPrimaryResourceBar()
+        {
+            var gauge = PluginInterface.ClientState.JobGauges.Get<DRGGauge>();
+
+            var barWidth = (BarWidth - xPadding * 2) / 2.0f;
+            var barSize = new Vector2(barWidth, BarHeight);
+            var xPos = CenterX - XOffset;
+            var yPos = CenterY + YOffset - 33;
```

Comment:
This 33 is a repeated value I notice, is there a constant that can be extracted here?